stream.go

```diff
@@ -386,8 +386,8 @@ func (s *stream) SetDeadline(t time.Time) error {
 }
 
 // CloseRemote makes the stream receive a "virtual" FIN stream frame at a given offset
-func (s *stream) CloseRemote(offset protocol.ByteCount) {
```

Comment:
The reason a `ByteCount` is used here, is that the H2 mapping in gQUIC requires this layer violation, which is why `CloseRemote` is not part of the public API.
This layer violation will be resolved in IETF QUIC